src/controllers/ajax/IndexAjaxController.php

```diff
@@ -220,15 +220,17 @@ protected function getActions(): array<string> {
     int $team_id,
     string $password,
   ): Awaitable<string> {
-    // Check if login is enabled
+    // Verify credentials first so we can allow admins to login regardless of the login setting
+    $team = await Team::genVerifyCredentials($team_id, $password);
+
+    // Check if login is disabled and this isn't an admin
     $login = await Configuration::gen('login');
-    if ($login->getValue() === '0') {
+    if (($login->getValue() === '0') &&
+        (!$team || (($team) && ($team->getAdmin() === false)))) {
```

Comment:
You can get rid of the redundant `($team)` I think. You should just be able to do: `$team === null || $team->getAdmin() === false`. I also changed `!$team` to an explicit null check, because I don't like mixing bools with nullable values.